src/test/regress/sql/multi_extension.sql

```diff
@@ -579,6 +577,19 @@ WHERE pgd.refclassid = 'pg_extension'::regclass AND
 	  pgio.schema    NOT IN ('pg_catalog', 'citus', 'citus_internal', 'test', 'columnar', 'columnar_internal')
 ORDER BY 1, 2;
 
+-- Snapshot of state at 11.2-2
+ALTER EXTENSION citus UPDATE TO '11.2-2';
+
+SELECT * FROM multi_extension.print_extension_changes();
+
+-- show running version
+SHOW citus.version;
+
+-- Test downgrade to 11.2-1 from 11.2-2
+ALTER EXTENSION citus UPDATE TO '11.2-1';
+
+DROP TABLE multi_extension.prev_objects, multi_extension.extension_diff;
+
```

Comment:
I think this portion should move earlier in the file. 
1) What if we mistakenly create some objects outside pg_catalog in this newly introduced version?
2) It is a good idea to keep all the version checks together.